vendor/google.golang.org/grpc/internal/transport/transport.go

```diff
@@ -374,7 +374,7 @@ func (s *Stream) TrailersOnly() bool {
 	return s.noHeaders
 }
 
-// Trailer returns the cached trailer metedata. Note that if it is not called
+// Trailer returns the cached trailer metadata. Note that if it is not called
```

Comment:
You should not directly modify the data in the vendor folder, this may cause CI to fail to pass